conan/api/conan_api.py

```diff
@@ -48,10 +50,12 @@ def __init__(self, cache_folder=None):
         init_colorama(sys.stderr)
         self.cache_folder = cache_folder or get_conan_user_home()
         self.home_folder = self.cache_folder  # Lets call it home, deprecate "cache"
-        self.migrate()
-
         # This API is depended upon by the subsequent ones, it should be initialized first
         self.config = ConfigAPI(self)
+        _check_conan_version(self)
+
+        self._helpers = self._Helpers(self)
```

Comment:
Sounds good, and as it is private I wouldn't worry that much about the name. Just maybe ``api_helpers = _ApiHelpers``, to make it a little less too generic?